docs/docs/addons-perf.md

```diff
@@ -57,6 +62,34 @@ The following methods use the measurements returned by [`Perf.getLastMeasurement
 
 * * *
 
+## Example Usage
+
+We will take simple example of a form which can be used to submit comments and then listing all comments. The example code is [available on GitHub](https://github.com/dhyey35/react-example-for-performance/blob/master/public/scripts/example.js).
+
+```javascript
+import { Component } from 'react'
+import Perf from 'react-addons-perf'
+
+const Comment = (props) => (
```

Comment:
We don't use this style of declaring components anywhere in the docs. We used `function` everywhere because it's easier to understand for people new to ES6.